Issue debugging with windows

**Describe the bug**
It may have been some time since we debugged on the command with windows.

To reproduce run this command in windows:

```bat
gradlew runApp -Popendcs.app=dbedit -Popendcs.profile=""C:\Users\ktarb\AppData\Roaming\.opendcs\usgs.profile" -Popendcs.debug=5005 -Pno.docs=true --info
```

Errors:

```
DLOG_DIR={C:\project\opendcs\install\build\run} -PC:\Users\ktarb\AppData\Roaming\.opendcs\edit-db-usgs.profile
Successfully started process 'command 'cmd''
'findstr' is not recognized as an internal or external command,
operable program or batch file.
```

runApp typo causes this error (windows only).

```
Caused by: java.util.regex.PatternSyntaxException: Illegal repetition near index 1
{C:/project/opendcs/install/build/run}/decodes.dbeditor.DecodesDbEditor.log.\d{4}-\d{2}-\d{2}.log
 ^
```

--- a/install/build.gradle
+++ b/install/build.gradle
@@ -162,7 +162,7 @@ task runApp(type: Exec) {
     println("Using app ${app}")
     if (osWin) {
         executable "cmd"
-        args "/c", binDir+app+ext, "-d3", "-DLOG_DIR={$runDir}"
+        args "/c", binDir+app+ext, "-d3", "-DLOG_DIR=${runDir}"
     } else {
         executable binDir+app
         args "-d3", "-DLOG_DIR=${runDir}"

